developer-extension/src/panel/components/json.tsx

```diff
@@ -314,7 +327,7 @@ function CopyMenuItem({ value, children }: { value: unknown; children: ReactNode
   return (
     <Menu.Item
       onClick={() => {
-        copy(JSON.stringify(value, null, 2))
+        copy(JSON.parse(JSON.stringify(value, null, 2)))
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ why is it parsed and then stringified?

Reply:
The stringified result has quotation marks which is not supported with our current search terms. Maybe it would be easier to read if I explicitly remove the quotation marks.

Reply:
yes, or adding a comment would do too

Reply:
This will break other `<CopyMenuItem` usages, because sometimes `value` is an object, so if we use `copy({})` we'll end up copying `[object Object]` (I guess).

What about something like this:

```suggestion
        if (typeof value === 'object') {
          copy(JSON.stringify(value, null, 2))
        } else {
          copy(String(value))
        }
```

It should cover your usecase without breaking other ones.